packages/opentelemetry-cloud-monitoring-exporter/test/monitoring.test.ts

```diff
@@ -243,7 +289,11 @@ describe('MetricExporter', () => {
       await meter.collect();
       const records2 = meter.getProcessor().checkPointSet();
 
-      await exporter.export(records2, () => {});
+      const result2 = await new Promise<ExportResult>(resolve => {
```

Comment:
I believe removing the async from the export function made these changes necessary.